src/App/integrationTests/App.syncUserDeniedPushErrors.test.js

```diff
@@ -39,7 +39,7 @@ test('User being denied push sync shows toasts', async () => {
   expect(project100ToastContent).toHaveTextContent(
     'You do not have permission to sync data to Project 100. Please check your notifications and consult with a project administrator about your project role.',
   )
-  expect(project100ToastContent).toHaveTextContent('Unsaved data:')
+  expect(project100ToastContent).toHaveTextContent("The following haven't been saved: ")
```

Comment:
I think we should not use contractions (applying to all such changes, but here: `have not`)